eth/chains/base.py

```diff
@@ -468,7 +468,9 @@ def import_block(self,
             )
 
         base_header_for_import = self.create_header_from_parent(parent_header)
-        block_result = self.get_vm(base_header_for_import).import_block(block)
+        # Make a copy of the clean header with some annotations, for richer logging in the VM
```

Comment:
This comment didn't make it any clearer to me why we need to make this copy.  My educated guess is that it gives access to the expected total gas used by this block which otherwise we wouldn't be able to access.